wasm.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+//
+// This is the main use to run the TLS interpreter with WASM.
+// The method tslToWarpScript is exported to the JavaScript.
+//
+
+import (
+	"./tsl"
+	"bytes"
+	"strings"
+	"syscall/js"
+)
+
+// tslToWarpScriptWasm method to generate WarpScript from TSL statements in WASM and return result with a callback
+func tslToWarpScriptWasm(this js.Value, inputs []js.Value) interface{} {
+	tslQuery := inputs[0].String()
+	defaulToken := inputs[1].String()
+	allowAuthenticate := inputs[2].Bool()
+	callback := inputs[len(inputs)-1:][0]
+
+	// Get query parsing result
+	parser, err := tsl.NewParser(strings.NewReader(tslQuery), "warp", defaulToken, 0, "", "")
```

Comment:
What could be great would be to set linecount (0), queryRange and samplers using user parameters

Reply:
Which parameters are needed? Can you provide me a list and where I need to use it?

Reply:
in NewParser: 
0 can be replace by a lineCount param
then just after the 0, the "" can be replace by a string queryRange param
then just after the "", the last "" can be replace by a string samplers param
The "warp" string won't work and you should put "warp10" instead ;)

Reply:
@NotBad4U do you see the change to do here? Otherwise I will merge this PR and do the update ;)